src/api/app/views/webui/request/_request_header.html.haml

```diff
@@ -75,7 +75,7 @@
     = render partial: 'actions_details', locals: { bs_request: bs_request, action: action,
                                                     prev_action: prev_action, next_action: next_action,
                                                     actions: actions, diff_to_superseded_id: diff_to_superseded_id,
-                                                    page_name: page_name }
+                                                    page_name: page_name, notification_id: current_notification }
```

Comment:
```suggestion
                                                    page_name: page_name, current_notification: current_notification }
```

Reply:
Inside the partial we use `current_notification`

Reply:
It's a shame the specs didn't catch this...

Reply:
We don't test views, unfortunately...

Reply:
Well, we do, but not all. I added a feature spec that at least walks the "add role" request type and catches the error